test/operator/operator_test.py

```diff
@@ -213,7 +191,9 @@ def test_linear_Op():
     x = np.random.rand(3)
     out = np.random.rand(3)
 
-    Aop = MultiplyOp(A)
+    r3 = odl.Rn(3)
+
+    Aop = MatVecOperator(r3, r3, A)
```

Comment:
Well having default values with optional selection is (imo) almost always better then forcing the users to always pick the obvious choice, see for example if users had to write `Rn(3, dtype=float)`.